shared/editor/lib/markdown/normalize.ts

```diff
@@ -18,5 +29,11 @@ export default function normalizePastedMarkdown(text: string): string {
   // find single newlines and insert an extra to ensure they are treated as paragraphs
   text = text.replace(/\b\n\b/g, "\n\n");
```

Comment:
After various tests I'm happy to just remove this line instead.

In the future it's best if you file issues before PRs